Cargo.toml

```diff
@@ -115,6 +115,10 @@ name = "hello_world"
 path = "examples/hello_world.rs"
 
 # 2D Rendering
+[[example]]
+name = "change_sprite_runtime"
+path = "examples/2d/change_sprite_runtime.rs"
```

Comment:
I think the name "modify_sprite" is probably slightly clearer to beginners? I'll defer to you and/or Cart though.